apps/web/src/components/Composer/NewPublication.tsx

```diff
@@ -214,6 +239,13 @@ const NewPublication: FC<NewPublicationProps> = ({ publication }) => {
       return;
     }
 
+    if (
+      showNewPublicationModal &&
+      modalPublicationType === NewPublicationTypes.Spaces
+    ) {
+      toast.success('Spaces created successfully!');
```

Comment:
we should add translations for this also?

Reply:
Updated